AppController/lib/infrastructure_manager_client.rb

```diff
@@ -184,6 +184,7 @@ def spawn_vms(num_vms, options, jobs, disks)
     parameters['zone'] = options['zone']
     parameters['region'] = options['region']
     parameters['IS_VERBOSE'] = options['verbose']
+    parameters['instance_type'] = options ['compute_instance_type']
```

Comment:
shouldn't this be associated with the jobs type? that is, if job == compute (or job.include?(compute) or whatever trickery ruby uses) then do the assignment of the type?

Reply:
So this spawn vms method is used only for autoscaling compute nodes right? So it just needs an instance type to be specified which it can use for autoscaling. At this point, I don't believe it would need another check to determine that. 

But I might need a check to see if it already has a default instance type specified outside of the ips layout, in which case it might override it. Will test that and update here.